src/components/DateTimePicker/Internal/ocpicker.module.scss

```diff
@@ -133,18 +133,18 @@
     }
 
     &-large &-header {
-        margin: 0 16px 16px 16px;
-        padding: 16px 0 12px 0;
+        margin: 0 $space-m $space-m $space-m;
```

Comment:
given shorthand notation, you can shorten these to `0 $space-m $space-m;` if desired